Sources/SpelledPitch/PitchSpeller/Wetherfield/WeightedGraphScheme.swift

```diff
@@ -21,3 +21,10 @@ struct WeightedGraphScheme <Node,Weight>: UndirectedGraphSchemeProtocol, Weighte
         return weight(Edge(start, end))
     }
 }
+
+extension WeightedGraphScheme {
+    
+    var directed : WeightedDirectedGraphScheme<Node,Weight> {
```

Comment:
Wanna `var directed: ` instead of `var directed : ` ?

Reply:
oops!